ups-connector-demo/src_hd/ups/connector/demo/Address/AddressData.ivyClass

```diff
@@ -0,0 +1,16 @@
+AddressData #class
+ups.connector.demo.Address #namespace
+requestoption Integer #field
+requestoption PERSISTENT #fieldModifier
+version String #field
+version PERSISTENT #fieldModifier
+regionalrequestindicator String #field
+regionalrequestindicator PERSISTENT #fieldModifier
+maximumcandidatelistsize Integer #field
+maximumcandidatelistsize PERSISTENT #fieldModifier
```

Comment:
this property name is not consistent with another name of this class, we have 2 styles inside 1 data class - camelcase and lowercase => please unify it. Thanks!